exporter/opentelemetry-exporter-otlp-proto-common/tests/test_log_encoder.py

```diff
@@ -0,0 +1,253 @@
+# Copyright The OpenTelemetry Authors
```

Comment:
Probably would need a test for metrics as well right?

Reply:
Yep, I will write one